packages/components/index.d.ts

```diff
@@ -33,7 +33,7 @@ declare module 'nebenan-react-hocs/lib/use_event_listener' {
 declare module 'nebenan-helpers/lib/utils' {
   export function invoke<TFunc>(
     fn: TFunc | undefined | null,
-    ...args: Paramters<TFunc>
+    ...args: Parameters<TFunc>
```

Comment:
Interesting how our linter and compiler didn't catch this 🤔

Reply:
looks like our current linting via "tsc --noEmit" doesn't catch errors in the index definition files at all. Try out by adding something stupid there in core -> nothing....

Maybe we have to switch to a "proper" linter sometime in the future?

Reply:
`tsc --noEmit` is mostly used for type-checking without outputting transpiled code 🤔 

There's a card on Favro about linters, so, we can discuss it then.